ui/client/components/graph/Graph.js

```diff
@@ -72,8 +78,30 @@ class Graph extends React.Component {
 		this.processGraphPaper.setDimensions(area.offsetWidth, area.offsetHeight)
 	}
 
+	copyNode(event) {
+		if (NodeUtils.noChosenNode(this.props.nodeToDisplay) || !this.props.allModalsClosed) {
+			return
```

Comment:
Hmm, I don't like empty return.. Mayby we can do this in better way?

Reply:
I like short returns but I can put writing to clipboard under the if statement since nothing else happens in this function.

Reply:
Done.